backend/src/db/migrations/20230820083309_add_user_details_table.ts

```diff
@@ -0,0 +1,43 @@
+import { type Knex } from 'knex';
+
+import { DatabaseTableName } from '#libs/packages/database/database.js';
+
+const TABLE_NAME = 'user_details';
+
+const ColumnName = {
+  ID: 'id',
+  USER_ID: 'user_id',
+  FULL_NAME: 'full_name',
+  AVATAR_ID: 'avatar_id',
+  SURVEY: 'survey',
+  CREATED_AT: 'created_at',
+  UPDATED_AT: 'updated_at',
+};
+
+function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable(TABLE_NAME, (table) => {
+    table.increments(ColumnName.ID).primary();
+    table.integer(ColumnName.USER_ID);
+    table
+      .foreign(ColumnName.USER_ID)
+      .references(ColumnName.ID)
+      .inTable(DatabaseTableName.USERS)
+      .onUpdate('CASCADE')
+      .onDelete('SET NULL');
```

Comment:
Create constants for strategies

Reply:
These constants are also not allowed to be imported? I'm asking because it won't be changed in future as I undersand